kinetic/distribution.yaml

```diff
@@ -8685,6 +8685,18 @@ repositories:
       version: master
     status: developed
   novatel_oem7_driver:
+    doc:
+      type: git
+      url: https://github.com/novatel/novatel_oem7_driver.git
+      version: 1.0.0
```

Comment:
This should be a branch, not a tag (so 'master' in this case).  Our CI usually catches this, but for some reason didn't this time.